prdoc/pr_4976.prdoc

```diff
@@ -0,0 +1,15 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Add pub to xcm::v4::PalletInfo
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      Forgot to make v4 PalletInfo fields public. Without them we cannot make use of the struct.
+
+crates:
+  - name: staging-xcm
+    bump: patch
+
```

Comment:
```suggestion
    validate: false
```
@Morganamilo please double check.

Reply:
The check is conceptually correct, the visibility of inner members of some public/crate-exported struct has changed. So strictly speaking `major` or at least `minor` bump is required.

But in this case we know better, this change is 100% backwards compatible, and bumping this only leads to unnecessary bumping of more downstream crates. So we do `patch` instead.